#存储锥桶xy信息，无顺序

float64 x1
float64 y1
float64 x2
float64 y2
float64 x3
float64 y3
float64 x4
float64 y4
float64 x5
float64 y5
float64 x6
float64 y6
